### PR TITLE
Dynamic labels

### PR DESCRIPTION
**Pull request by linusnorton**
Add dynamic labels that switch the content in XUI for joint/sole applications and divorce/dissolutions.

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/divorce/divorcecase/model/LabelContent.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        if (divorceOrDissolution != null && divorceOrDissolution.isDivorce()) {
+            unionType = "divorce";
+            unionTypeUC = "Divorce";
+        } else {
+            unionType = "dissolution";
+            unionTypeUC = "Dissolution";
+        }
+    }
+
+    public void setApplicationTYpe(ApplicationType applicationType) {
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/divorce/divorcecase/model/LabelContent.java:46]: TYpe ->Type

File: src/main/java/uk/gov/hmcts/divorce/divorcecase/model/LabelContent.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    private String theApplicant2UC;
+
+    @CCD(label = "Applicant or Respondent")
+    private String applicant2UC;
+
+    @CCD(label = "divorce or dissolution")
+    private String unionType;
+
+    @CCD(label = "Divorce or Dissolution")
+    private String unionTypeUC;
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/divorce/divorcecase/model/LabelContent.java:34]: what does UC signifies ?

File: src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Application.java [hunk 3]
```diff
@@ -36,7 +36,7 @@
     private static final int SUBMISSION_RESPONSE_DAYS = 14;
 
     @CCD(
-        label = "Has applicant 1's marriage broken down irretrievably?",
+        label = "Has the applicant's marriage broken down irretrievably?",
```
Comment (linusnorton) [inline src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Application.java:39]: this one is just an unrelated content error

**Review by nitinprabhu — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/divorce/divorcecase/model/LabelContent.java (hunk 2 above)
Comment (linusnorton) [reply src/main/java/uk/gov/hmcts/divorce/divorcecase/model/LabelContent.java:34]: first character upper case

**Pull request merged**